types/office-js-preview/index.d.ts

```diff
@@ -3905,6 +3905,17 @@ declare namespace Office {
          *   </tr>
          * </table>
          * 
+         * **Type-specific behaviors**
+         * 
+         * <table>
+         *   <tr>
+         *     <td>`Office.CoercionType.XmlSvg`</td>
+         *     <td>There is a 60k size limitation for SVG insertions.</td>
```

Comment:
If this is 60k as in 60kB, perhaps we should note that.
```suggestion
         *     <td>There is a 60kB size limitation for SVG insertions.</td>
```